homeassistant/components/weather/openweathermap.py

```diff
@@ -129,10 +131,22 @@ def attribution(self):
         """Return the attribution."""
         return ATTRIBUTION
 
+    @property
+    def forecast(self):
+        """Return the forecast array."""
+        temp_unit = ('celsius' if self.hass.config.units.is_metric else
+                     'fahrenheit')
+
+        return [{'datetime': entry.get_reference_time('iso'),
+                 'temp': entry.get_temperature(temp_unit).get('temp', None)}
```

Comment:
Good! Make them constants in the component and import them to this module. You might also be able to reuse attributes (temperature) that are already defined in the const module. Not sure if that would be appropriate here though.